ci/run-sanity.sh

```diff
@@ -5,9 +5,26 @@ cd "$(dirname "$0")/.."
 # shellcheck source=multinode-demo/common.sh
 source multinode-demo/common.sh
 
+if [[ -z $CI ]]; then
+  # Build eargerly if needed for local development. Otherwise, odd timing error occurs...
+  $solana_keygen --version
+  $solana_genesis --version
+  $solana_faucet --version
+  $solana_cli --version
+  $agave_validator --version
+  $solana_ledger_tool --version
+fi
+
 rm -rf config/run/init-completed config/ledger config/snapshot-ledger
 
-SOLANA_RUN_SH_VALIDATOR_ARGS="--full-snapshot-interval-slots 200" timeout 120 ./scripts/run.sh &
+# Sanity-check that agave-validator can successfully terminate itself without relying on
+# process::exit() by extending the timeout...
+# Also the banking_tracer thread needs some extra time to flush due to
+# unsynchronized and buffered IO.
```

Comment:
> > i think upper bounds set by preceding pipeline steps (quic & sigverify) is enough?
> 
> Are you saying the extra time needed for the banking_tracer thread to flush is dictated by the previous stages ?

yes for production environments. the banking_tracer thread is very lightweight because `Packet` serialization is cheap. So, it won't grow its buffer indefinitely because quic & sigverify is always slower.

> and even if that is true, do you have an idea how much time this is in actual seconds ?

it churned 1GB/sec when i benched long time ago, iirc. And banking_tracer is 100ms interval. so, i guess most are flushed to dirty kernel page caches within 50ms (avg interval) + 20-50ms.

however, in the context of this `ci/run-sanity.sh` test, it takes some arbitrarily longer time because this is built with dev profile.